editor/editor_node.cpp

```diff
@@ -3366,14 +3365,14 @@ void EditorNode::set_addon_plugin_enabled(const String &p_addon, bool p_enabled,
 
 	EditorPlugin *ep = memnew(EditorPlugin);
 	ep->set_script(script);
-	plugin_addons[p_addon] = ep;
+	name_to_editor_plugin[p_addon] = ep;
```

Comment:
`name_to_editor_plugin` doesn't sound right. The original name is ok, not sure how it can be improved.

Reply:
The "addon_plugin" name in the method itself is a bit weird. Doesn't `addon` and `plugin` mean the same in this context? Or is there an API for non-plugin addons?

Since this stores editor plugins, it could just be `editor_plugins`.

Reply:
The point of this rename was that "addon" is inconsistent which the usual "plugin" naming, but just changing "addons" to "plugins" wont work since there is already `Vector<EditorPlugin *> editor_plugins;` 
In addition, "plugin_addons" sound like it is just a list of all editor plugins/addons, which is not the case: It is actually of the type Map<String, EditorPlugin *>, so it maps the name of the editor plugin to the editor plugin object. In my opinion naming a map x_to_y makes it clearer since it directly tells you what is mapped to what.

Reply:
If I understand this variable correctly, it tracks which plugin was added from which user addon.

Reply:
Ok, this naming is indeed a bit weird. "Addon plugins" are basically editor plugins provided by the user, right? Should we then rename the corresponding methods/variables to use a "user_plugin" name scheme instead of a mix of plugin_addons and addon_plugins?

Reply:
I don't know tbh. It's rather established that installable plugins are called addons (their default folder is even called that). "addon plugins" could indeed be renamed to "user plugins", but the other usage of "addon" is probably fine.